python/langsmith/evaluation/_runner.py

```diff
@@ -82,6 +82,7 @@ def evaluate(
     experiment_prefix: Optional[str] = None,
     description: Optional[str] = None,
     max_concurrency: Optional[int] = None,
+    num_repetitions: Optional[int] = 1,
```

Comment:
What's the difference between none and 1?

Reply:
Seems like it shouldn't be nullable?

Reply:
ya fair, good point

Reply:
done